manifest.json

```diff
@@ -61,7 +61,7 @@
     {
       "id": "meta_languages",
       "type": "meta",
-      "version": "0.1.4",
+      "version": "0.1.5",
```

Comment:
Don't worry about this. It's mostly to keep track of deps changes.

Reply:
Okay, I've reverted it.